packages/eslint-plugin-x/lib/configs/eslint.js

```diff
@@ -23,6 +23,48 @@ module.exports = {
       'require-atomic-updates': 'error',
       'require-await': 'error',
       strict: ['error', 'global']
-    }
+    },
+    overrides: [
+      {
+        files: ['src/**/*.vue'],
+        rules: {
+          /*
+           * We use 'vue/max-len' that "eslint-plugin-vue" has. This rule understands much better
+           * the ".vue" files.
+           * It helps us in cases that the "<style>" tag's section has long lines of
+           * code.
+           * (e.g: "@include foo(background-color, color, $selector: '#{$component-class}__kpi');
+           * https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/max-len.md
+           */
+          'vue/max-len': [
+            'error',
+            {
+              ignorePattern: '|class=".*"$ |https://github',
```

Comment:
This `ignorePattern` config was previously set in [x-components' eslint config](https://github.com/empathyco/x/blob/main/packages/x-components/.eslintrc.js#L22).